### ISSUE TITLE
Angular 5 incorrect peer dependency

### ISSUE THREAD

**Issue by Schwobaland** [Alberthaff/ngx-papaparse#10]
We are running the latest Angular release 5.1.0 and getting the following error.
```
ngx-papaparse@2.0.2" has incorrect peer dependency "@angular/core@^4.1.2".
```
Can you loosen the dependency range, so we can get rid of the warning?

**Comment by Alberthaff** [comment]
Thank you for reporting the issue.
The issue has been fixed as of version 2.0.3

**Comment by Schwobaland** [comment]
tried it just now - and warning is gone :) Thank you! 👍
